app/views/content/help-and-advice.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Help and advice
+navigation: 50
+---
+
+We’re here to answer your questions and provide free advice about teacher training, whether you’re just thinking about teaching or you’re ready to apply.
+
+##Come to an event
+
+Our events are a great way to meet people and talk about any concerns or questions you have about becoming a teacher.
+
+###Train to Teach
+
+Train to Teach events are free to attend and you'll be able to:
+
+- put your questions to expert advisers, teachers and training providers
+- chat with current teachers
+- find out what a career in teaching is really like
+- watch presentations which provide detailed guidance on how to get into teaching, the application process and funding your training
+
+
+###Online question and answer sessions
+
+There are online question and answer sessions where you can ask a panel of specialists the questions that matter to you. These sessions cover a range of different topics so you can ask specific questions and receive advice tailored to your particular circumstances.
```

Comment:
Do we want to say that these are run by the Department of Education? We did this in the new events design based on UR (@KeishaDfE can confirm) and also with a view to differentiating these events.

Reply:
@gemmadallmandfe yes, I'd make that clear here too.
I still wonder whether users think this is a zoom event where you can ask questions to a panel rather than a text based q&a event on the GiT website

Reply:
I think that's a really good point @KeishaDfE - we should make it clear that you are posting questions (text-based)